examples/pbfhr/multi_pt/prt_2ref/input_10s.py

```diff
@@ -1,206 +0,0 @@
-''' changed from the initial input file
```

Comment:
Deleting the failing files seems counterproductive. Maybe we can talk about backwards compatibility.